kettle/Makefile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 IMG = gcr.io/k8s-testimages/kettle
-TAG := $(shell date +v%Y%m%d)-$(shell git describe --tags --always --dirty)
```

Comment:
the TAG also includes the commit so even if you build many images they would have different commits right?
unless you are building and pushing from local changes in which case they would have the dirty tag and fine to be overwritten.

Reply:
The image will be overwritten but if you try to re-pull from `kubectl set image` it will think it already has the correct image. Is there a force?

Reply:
`imagePullPolicy: Always` does not seem to impact `set image`

Reply:
not sure what you mean, wouldn't a new image have a new tag if it's based off of a new commit?
why will it think it already has the correct image?

Reply:
Here is the process that was bugging me:
- make a change on a `dirty` branch
- upload change with tag: `123abc`
- `kubectl set image kettle:<image>:v123abc`
- check, something was broken.
- make fix and re-upload (this will have the same tag `123abc`
- `kubectl set image kettle:<image>:v123abc` <<< This now does nothing because it doesn't try to repull

Reply:
ok so this is for local testing, not prod manifests.
can you just push it with a different tag?

Reply:
I could force the tag, or what I have been doing is setting it to another image than switching back